providers/openrouter/models/x-ai/grok-4-fast:free.toml

```
@@ -0,0 +1,23 @@
+name = "Grok 4 Fast (free)"
+release_date = "2025-08-19"
+last_updated = "2025-08-19"
+attachment = false
+reasoning = true
+temperature = true
+knowledge = "2024-11"
+tool_call = true
+open_weights = false
+
+[cost]
+input = 0.00
+output = 0.00
+cache_read = 0.00
+cache_write = 0.00
+
+[limit]
+context = 2_000_000
+output = 2_000_000
+
+[modalities]
+input = ["text", "image"]
+output = ["text", "image"]
```

image output [seems not supported](https://docs.x.ai/docs/models/grok-4-fast-reasoning).

```suggestion
output = ["text"]
```